code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -563,6 +583,227 @@
+		if(4)
+			new /obj/item/gun/magic/staff/healing(src)
+			value += 1
+		if(5)
+			new /obj/item/gun/magic/staff/chaos(src)
+			value += 2
+		if(6)
+			new /obj/item/gun/magic/staff/animate(src)
+			value += 2
+		if(7)
+			new /obj/item/gun/magic/staff/focus(src)
+			value += 2 //Probably value 2? Test it out
+		if(8)
+			new /obj/item/gun/magic/hook(src)// Basically a staff
+			value += 1
+		if(9)
+			new /obj/item/hierophant_club(src)
+			value += 3 //Very strong, very point intensive. Will be rare.
+		if(10)
+			new /obj/item/lava_staff(src)
+			value += 2 //Hot seller
+
+	//Random magical artifact.
+	switch(rand(1, 9))
+		if(1)
+			new /obj/item/necromantic_stone(src)
+			value += 2
+		if(2)
+			new /obj/item/scrying(src)
+			value += 1 //Thematic discount
+		if(3)
+			new /obj/item/organ/internal/heart/cursed/wizard(src)
+			value += 1
+		if(4)
+			new /obj/item/organ/internal/vocal_cords/colossus/wizard(src)
+			value += 2
+		if(5)
+			new /obj/item/warp_cube/red(src)
+			value += 1
+		if(6)
+			new /obj/item/reagent_containers/food/drinks/everfull(src)
+			value += 2
+		if(7)
+			new /obj/item/clothing/suit/space/hardsuit/shielded/wizard(src)
+			value += 2
+		if(8)
+			new /obj/item/jacobs_ladder(src) //This is still in the code? Well, why not.
+			value += 1
+		if(9)
+			new /obj/item/immortality_talisman(src) //Spells recharge during this, though minute cooldown. Potentially useful
+			value += 1
+
+	//Summon
+	switch(rand(1, 8))
+		if(1)
+			new /obj/item/antag_spawner/slaughter_demon(src)
+			value += 2
+		if(2)
+			new /obj/item/antag_spawner/morph(src)
+			value += 1
+		if(3)
+			new /obj/item/antag_spawner/slaughter_demon/laughter(src)
+			value += 1
+		if(4)
+			new /obj/item/antag_spawner/slaughter_demon/shadow(src)
+			value += 1
+		if(5)
+			new /obj/item/antag_spawner/revenant(src)
+			value += 1
+		if(6)
+			new /obj/item/contract(src)
+			value += 2
+		if(7)
+			new /obj/item/guardiancreator(src)
+			value += 1
+		if(8)
+			if(prob(25))
+				new /obj/item/reagent_containers/food/snacks/grown/nymph_pod(src)
+				new /obj/item/slimepotion/sentience(src)
+			else
+				new /obj/item/paicard(src) //Still useful, not a point useful.
+
+	//Treat / potion. Free.
+	switch(rand(1, 5))
+		if(1)
+			new /obj/item/storage/box/syndidonkpockets(src) // Healing + speed
+		if(2)
+			new /obj/item/reagent_containers/food/drinks/bottle/dragonsbreath(src) // Killing
+		if(3)
+			new /obj/item/reagent_containers/food/drinks/bottle/immorality(src) // Super healing for 20 seconds
+		if(4)
+			new /obj/item/reagent_containers/food/snacks/meatsteak/stimulating(src) //Healing + stun immunity
+		if(5)
+			new /obj/item/reagent_containers/food/snacks/plum_pie(src) // Great healing over long period of time
+
+	if(value > MAX_VALUE || value < MIN_VALUE)
+		if(attempts >= 5)
+			message_admins("Failed to generate the wizard a properly priced magic nanny bag!")
+		else
+			new /obj/item/storage/backpack/duffel/magic_nanny_bag(get_turf(loc), attempts += 1)
```

Comment:
```suggestion
			new /obj/item/storage/backpack/duffel/magic_nanny_bag(get_turf(loc), attempts + 1)
```
don't need `+=` because attempts doesn't need to be assigned the new value cause you're just passing it as a param